api/types/server_test.go

```diff
@@ -562,34 +493,12 @@ func TestServerCheckAndSetDefaults(t *testing.T) {
 				},
 			},
 			assertion: func(t *testing.T, s *ServerV2, err error) {
-				require.EqualError(t, err, `AWS OIDC Integration is required for "openssh-ephemeral-key" SubKind`)
+				require.EqualError(t, err, `missing AWS OIDC Integration (required for "openssh-ephemeral-key" SubKind)`)
 			},
 		},
 		{
-			name: "valid OpenSSHEphemeralKey node",
-			server: &ServerV2{
-				Kind:    KindNode,
-				SubKind: SubKindOpenSSHEphemeralKeyNode,
-				Version: V2,
-				Metadata: Metadata{
-					Name:      "5da56852-2adb-4540-a37c-80790203f6a9",
-					Namespace: defaults.Namespace,
-				},
-				Spec: ServerSpecV2{
-					Addr:     "example:22",
-					Hostname: "openssh-node",
-					CloudMetadata: &CloudMetadata{
-						AWS: &AWSInfo{
-							AccountID:        "123456789012",
-							InstanceID:       "i-123456789012",
-							Region:           "us-east-1",
-							AvailabilityZone: "us-east-1b",
-							VPCID:            "vpc-abcd",
-							Integration:      "teleportdev",
-						},
-					},
-				},
-			},
+			name:   "valid OpenSSHEphemeralKey node",
+			server: makeOpenSSHEphemeralKeyNode(func(_ *ServerV2) {}),
```

Comment:
```suggestion
			server: makeOpenSSHEphemeralKeyNode(func(*ServerV2) {}),
```

We could also make it work with `nil`. Up to you.